content/app/development/ux/texts/_index.en.md

```diff
@@ -324,12 +324,36 @@ I `App/config/texts/resource.en.json`:
   "language": "en",
   "resources": [
     {
-      "id": "ServiceName",
+      "id": "appName",
       "value": "Auto deploy application"
     },
     .
     .
     .
   ]
 }
-```
\ No newline at end of file
+```
+
+## Endre applikasjonseier tekst
+
+I applikasjonen så vises applikasjonsnavn og applikasjonseier-tekstene øverst i skjema.
+
+![Tekster i appen](app-name-app-owner.png "Appnavn og appeier tekster")
+
+Applikasjonsnavn hentes som standard ut fra tekstene som er definert i [altinn-orgs.json](https://github.com/Altinn/altinn-cdn/blob/master/orgs/altinn-orgs.json).
+Om det er ønskelig å endre på dette navnet kan det gjøres ved å legge til nøkkelen `appOwner` i tekstressursene. Denne vil da overstyre det som ligger på CDN.
+
+Eksempel:
+```json
+{
+  "language": "en",
+  "resources": [
+    {
+      "id": "appOwner",
+      "value": "Test Ministry"
+    },
+    .
+    .
+    .
+  ]
+}
```

Comment:
```suggestion
## Change app owner text

In the application, the app name and owner is displayed in the top of the form.

![Texts in the app](app-name-app-owner.png "App name and app owner texts")

App name is fetched by default from the texts defined in [altinn-orgs.json](https://github.com/Altinn/altinn-cdn/blob/master/orgs/altinn-orgs.json).
If you want to change this name, you can do that by adding the key `appOwner` in the text resources. This will take priority over the values gotten from CDN.

Example:
```json
{
  "language": "en",
  "resources": [
    {
      "id": "appOwner",
      "value": "Test Ministry"
    },
    .
    .
    .
  ]
}
```
```suggestion
## Endre applikasjonseier tekst

I applikasjonen så vises applikasjonsnavn og applikasjonseier-tekstene øverst i skjema.

![Tekster i appen](app-name-app-owner.png "Appnavn og appeier tekster")

Applikasjonsnavn hentes som standard ut fra tekstene som er definert i [altinn-orgs.json](https://github.com/Altinn/altinn-cdn/blob/master/orgs/altinn-orgs.json).
Om det er ønskelig å endre på dette navnet kan det gjøres ved å legge til nøkkelen `appOwner` i tekstressursene. Denne vil da overstyre det som ligger på CDN.

Eksempel:
```json
{
  "language": "en",
  "resources": [
    {
      "id": "appOwner",
      "value": "Test Ministry"
    },
    .
    .
    .
  ]
}
```